apps/api/src/routes/v1/dashboardData.ts

```diff
@@ -144,22 +144,65 @@ const route = createRoute({
 });
 
 export default new OpenAPIHono().openapi(route, async c => {
-  const chainStatsQuery = await getChainStats();
-  const dashboardData = await getDashboardData();
-  const networkCapacity = await getNetworkCapacity();
-  const networkCapacityStats = await getProviderGraphData("count");
-  const latestBlocks = await getBlocks(5);
-  const latestTransactions = await getTransactions(5);
 
-  const chainStats = {
-    height: latestBlocks[0].height,
-    transactionCount: latestBlocks[0].totalTransactionCount,
-    ...chainStatsQuery
-  };
+  let chainStats;
+  let now;
+  let compare;
+  let networkCapacity;
+  let networkCapacityStats;
+  let latestBlocks;
+  let latestTransactions;
+
+  try {
+    const chainStatsQuery = await getChainStats();
+    chainStats = { ...chainStatsQuery };
+  } catch (error) {
+    chainStats = {}
+    console.error("Failed to fetch getChainStats:", error);
```

Comment:
Please use the new logger instead of `console.error`. You can instantiate it at the top of the file like so:
`const logger = new LoggerService({ context: "DashboardDataEndpoint" });`

Reply:
It would also be a good idea to send those exceptions to Sentry.
```
import * as Sentry from "@sentry/node";
```
```
Sentry.captureException(error);
```